### PR TITLE
{lang}[GCCcore/6.4.0] Python v2.7.13

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)
depends on PR #4913

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.13-GCCcore-6.4.0-bare.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+osdependencies = [
+    # rely upon distribution for timely security updates
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+# We hide this by default since users should not use it in production,
+# high-performance Python should be delivered at compiler level with
+# a default extension set
+#hidden = True
```
Comment (verdurin) [inline easybuild/easyconfigs/p/Python/Python-2.7.13-GCCcore-6.4.0-bare.eb]: Please address the style errors reported by Travis:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/p/Python/Python-2.7.13-GCCcore-6.4.0-bare.eb:42:1: E265 block comment should start with '# '
```

File: easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# eg. http://www.sqlite.org/2017/sqlite-autoconf-3190300.tar.gz
+source_urls = ['http://www.sqlite.org/2017/']
+version_str = '%%(version_major)s%s00' % ''.join('%02d' %
+              int(x) for x in version.split('.')[1:])
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb]: Another style error:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb:28:15: E128 continuation line under-indented for visual indent
```

**Comment by verdurin** [comment]
@JackPerdue - also, why the duplication of `SQLite` and `Tcl`?

**Comment by JackPerdue** [comment]
@verdurin "also, why the duplication of SQLite and Tcl?"    @boegel mentioned on slack that if I have a new PR that depends on stuff from PRs already submitted but not merged yet to just include the (duplicates) and cite the original PR

File: easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb (hunk 2 above)
Comment (verdurin) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb]: @JackPerdue please remove this file, now that I've merged the separate `SQLite` PR

**Comment by JackPerdue** [comment]
[j-perdue@terra2 2017b]$ eb --check-style Python-2.7.13-GCCcore-6.4.0-bare.eb.terra 
== temporary log file in case of crash /tmp/j-perdue/tmp/eb-CRRlD1/easybuild-lZiD7s.log
Running style check on 1 easyconfig(s)...
[PASS] /general/sw/eb/hprc/ec-dev/2017b/Python-2.7.13-GCCcore-6.4.0-bare.eb.terra
All style checks passed!
== Temporary log file(s) /tmp/j-perdue/tmp/eb-CRRlD1/easybuild-lZiD7s.log* have been removed.
== Temporary directory /tmp/j-perdue/tmp/eb-CRRlD1 has been removed.
[j-perdue@terra2 2017b]$

Works here.

Not sure what Travis's problem is.

**Comment by verdurin** [comment]
@JackPerdue I don't know why Travis is complaining either.

Might have to wait until @boegel returns, or we can find someone with superior Travis knowledge.

**Comment by JackPerdue** [comment]
New version submitted as PR #4962    

I can't seem to use --update-pr without it wanting to drag in SQLite and TCL again... perhaps due to the way I removed them from this PR (deleting from branch in GitHub web thingie).  In any case, see the new one (tried to remove all the backslashes to make Travis happy).

**Comment by JackPerdue** [comment]
Travis seems much happier with #4962... closing this one.

**Pull request closed without merging**